autogen/agentchat/groupchat.py

```diff
@@ -537,7 +539,7 @@ def select_speaker(self, last_speaker: Agent, selector: ConversableAgent) -> Age
         # auto speaker selection with 2-agent chat
         return self._auto_select_speaker(last_speaker, selector, messages, agents)
 
-    async def a_select_speaker(self, last_speaker: Agent, selector: ConversableAgent) -> Agent:
```

Comment:
Thanks for the changes to `select_speaker` and `auto_select_speaker` for keeping the `selector` to support existing code bases and as a fall-back. Same can be applied to the async versions `a_select_speaker` and `a_auto_select_speaker`.